buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -225,35 +656,44 @@
       "value": null,
       "isSecret": true
     },
-    "PB_VsoRepoUrl": {
-      "value": "--branch $(PB_Branch) https://$(PB_VsoAccountName):$(PB_VsoPassword)@devdiv.visualstudio.com/DevDiv/_git/DotNet-Core-Setup-Trusted"
-    },
     "PB_VsoAccountName": {
       "value": "dn-bot"
     },
     "PB_VsoPassword": {
       "value": null,
       "isSecret": true
     },
+    "PB_Branch": {
```

Comment:
I don't have a strong attachment to it, I was just pointing out what it was previously used for. I think our quick conversation cycle on this PR seems to be causing a bit of static.

I agree with cleaning this up, I think that should be a separate PR though.  I just updated the value in this PR because it was clearly wrong.